source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/height-updater.js

```diff
@@ -0,0 +1,8 @@
+﻿(function ($, ko) {
+    ko.bindingHandlers.heightUpdater = {
+        update: function (element) {
+            $(element).height(1);
+            $(element).height(25 + element.scrollHeight);
```

Comment:
Since you are using `scrollHeight` without jQuery, what do you think about to completely remove jQuery dependency for this custom binding?